common4.py

```diff
@@ -25,14 +19,15 @@ def run_standard_text_flow(get_intervals, length_needed, total_num_pitches, make
         file.write("\nPossible pitch outcomes using {} for the {} entered intervals: {}\n\n".format(decision, seq_length, interval_set))
         file.write(str(string_list))
         file.write("\n\n")
-        if decision.lower() == 'y' or decision.lower() == 'yes':
+        if show_binary.lower().strip() in {'y', 'yes'}:
+            file.write("\n\nSelected binary ordering was: {} which means {}\n".format(binary_order, decision))
             file.write("\nbinary sequence\n[0=up, 1=down]\n")
             file.write(str(perm_list))
             file.write("\n\n")
         file.write("\nPitch tally:\n")
         file.write(str(total_tally))
         file.write("\nTotal # of piches: {}\n".format(total))
-        file.write(str(footer))
+        file.write(str(foot))
```

Comment:
`footer` is a function
`foot` is the variable you assigned to the response of that function

maybe renaming `footer` to something verb-oriented like `get_footer` or `make_footer` would prevent confusion like that